.gitignore

```diff
@@ -12,3 +12,4 @@ velocity.log
 /.metadata/
 /.recommenders/
 .factorypath
+bin/
```

Comment:
why do you want us to ignore bin/ ? Its not so uncommon to add some tooling into a bin directory.

Reply:
eclipse uses it for some state